common/descriptor_checksum.h

```diff
@@ -0,0 +1,11 @@
+#ifndef LIGHTNING_COMMON_DESCRIPTOR_CHECKSUM_H
+#define LIGHTNING_COMMON_DESCRIPTOR_CHECKSUM_H
+#include "config.h"
+
+/* https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md#reference */
+#define TOOL_DESCRIPTOR_CHECKSUM_LENGTH 8
+
+int descriptor_checksum(const char *descriptor, int desc_size, char *checksum,
```

Comment:
Apart from taking bitcoind's implementation (which is virtually this) and asserting they are equel in the fuzz target, i don't see how there could so not specifically for the checksum result.
However the test added later on would fail to import the descriptor if the checksum was incorrect, so there is already that.